docs/detections/api/rules/rules-api-create.asciidoc

```diff
@@ -583,6 +588,50 @@ The following variables can only be passed if the rule’s action frequency is f
 * `{{alert.id}}`: ID of the alert that scheduled actions for the rule
 * `{{alert.flapping}}`: A flag on the alert that indicates whether the alert status is changing repeatedly
 
+[[response-actions-object-schema]]
+===== `response actions` schema
+
+All fields are required:
+
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description
+
+|action_type_id |String a|The response action you want to add to a rule.
+
+* `.osquery`
+* `.endpoint`
++
+NOTE: To learn more about the requirements for using the isolate endpoint response action, refer to <<host-isolation-ov>>. 
++
+IMPORTANT: Host isolation involves quarantining a host from the network to prevent further spread of threats and limit potential damage. Be aware that automatic host isolation from a detection rule execution could result in unintended consequences, such as disrupting legitimate user activities or blocking critical business processes.
+
+|params |Object a|Object containing the allowed response action fields, which varies according to the response action.
+
+*Osquery* 
+
+For Osquery (`.osquery`), use a single query, a saved query, or a query pack:
+
+* `query` (string, optional): To run a single query, use the `query` field and enter a  SQL query. Example: `"query": "SELECT * FROM processes;"` 
+* `saved_query_id` (string, optional): To run a saved query, use the `saved_query_id` field and specify the saved query ID. Example: `"saved_query_id":  "processes_elastic"`
+* `packId` (string, optional): To specify a query pack, use the `packId` field. Example: `"packId": "processes_elastic"`
+* `ecs_mapping` (object, required): Map Osquery results columns or static values to Elastic Common Schema (ECS) fields. Example: `"ecs_mapping": {"process.pid": {"field": "pid"}`
```

Comment:
I noticed there are two open brackets { but only one close bracket. Just want to double-check this is right

Reply:
Yep! You're right, I need to close the second bracket -- only one field is being mapped. Thanks!

```suggestion
* `ecs_mapping` (object, required): Map Osquery results columns or static values to Elastic Common Schema (ECS) fields. Example: `"ecs_mapping": {"process.pid": {"field": "pid"}}`
```